data/creaturescripts/scripts/boss.lua

```diff
@@ -0,0 +1,229 @@
+local function pushSeparated(buffer, sep, ...)
+	local argv = {...}
+	local argc = #argv
+	for k, v in ipairs(argv) do
+		table.insert(buffer, v)
+		if k < argc and sep then
+			table.insert(buffer, sep)
+		end
+	end
+end
+
+local function insertItems(buffer, info, parent, items)
+	local start = info.running
+	for _, item in ipairs(items) do
+		if _ ~= 1 or parent > 100 then
+			table.insert(buffer, ",")
+		end
+
+		info.running = info.running + 1
+		table.insert(buffer, "(")        
+		pushSeparated(buffer, ",", info.playerGuid, parent, info.running, item:getId(), item:getSubType(), db.escapeBlob(item:serializeAttributes()))
+		table.insert(buffer, ")")
+
+		if item:isContainer() then
+			local size = item:getSize()
+			if size > 0 then              
+				local subItems = {}
+				for i = 1, size do
+					table.insert(subItems, item:getItem(i - 1))
+				end
+
+				insertItems(buffer, info, info.running, subItems)
+			end
+		end
+	end
+	return info.running - start
+end
+
+local function insertRewardItems(playerGuid, timestamp, itemList)
+	db.asyncStoreQuery('SELECT `pid`, `sid` FROM `player_rewards` WHERE player_id = ' .. playerGuid .. ' ORDER BY `sid` ASC;', 
```

Comment:
pid and sid are already used in [player_depotitems](https://github.com/otland/forgottenserver/blob/master/schema.sql#L271), player_inboxitems and player_items. I'm just following the current standard.